platform/linux-generic/m4/odp_ipsec_mb.m4

```diff
@@ -0,0 +1,19 @@
+#########################################################################
+# Check for libIPSec_MB availability
+#########################################################################
+ipsecmb_support=no
+AC_CHECK_HEADERS([ipsec-mb.h],
+        [AC_CHECK_LIB([IPSec_MB], [init_mb_mgr_auto], [ipsecmb_support=yes],
+                [ipsecmb_support=no])],
+        [ipsecmb_support=no])
+
+AS_IF([test "x$with_crypto" = "xipsecmb" -a "x$ipsecmb_support" = "xno"],
+      [AC_MSG_ERROR([IPSec MB library not found on this platform])])
+
+if test "$ipsecmb_support" = "yes"; then
+        IPSEC_MB_LIBS="-lIPSec_MB"
```

Comment:
This matches even if IPsec MB crypto is not enabled. `"x$with_crypto" = "xipsecmb"` should work as the previous check already fails if crypto lib is not available.

Reply:
Yes, thanks for your comments. This is wrong and corrected in latest commit.